static/src/javascripts/boot.js

```diff
@@ -40,6 +41,21 @@ const go = () => {
         const pubData: { browserId?: ?string, pageViewId?: ?string } =
             { browserId, pageViewId };
 
+
+        let recordedConsentTime = false;
+        onConsentChange(() => {
+            if (!recordedConsentTime) {
+                markTime('Consent acquired');
+                trackPerformance(
```

Comment:
If the 1% sample resolution is enough for you, great, otherwise [because of this](https://github.com/GoogleChrome/web-vitals/issues/80#issuecomment-714125270) we send Core Vitals as 'Events' instead of 'Timings' https://github.com/guardian/frontend/blob/126dfcbc1aa961650b7f7ff41ee50a12782bb62e/static/src/javascripts/projects/common/modules/analytics/google.js#L154

Just something to be aware of!

Reply:
ah ha - yeah i think it should be for now